Modelica/Electrical/Analog/Lines/M_OLine.mo

```diff
@@ -26,6 +26,8 @@ public
     each final min=Modelica.Constants.small,
     each unit="F/m") = {2.38e-11,1.01e-10,8.56e-11,5.09e-12,2.71e-11,2.09e-11,
     7.16e-11,1.83e-11,1.23e-10,2.07e-11} "Capacitance per meter";
+  parameter Boolean useInternalGround=true "Default: true = use internal ground / false = use reference pin"
```

Comment:
```suggestion
  parameter Boolean useInternalGround=true "Use internal ground if true / use reference pin if false"
```

Reply:
So far we did not use the spelling `Default:` in other parameter doc strings, so I suggest to omit it.

Reply:
Proposed style is:

```mo
parameter Boolean useInternalGround=true "= true if internal ground is used, otherwise usereference pin";
```